ratelimit/redis.go

```diff
@@ -174,7 +174,19 @@ func (c *clusterLimitRedis) allow(ctx context.Context, clearText string) (bool,
 	return true, nil
 }
 
-// Allow is like AllowContext, but not using a context.
+// Allow returns true if the request calculated across the cluster of
+// skippers should be allowed else false. It will share it's own data
+// and use the current cluster information to calculate global rates
+// to decide to allow or not.
+//
+// Performance considerations:
+//
+// In case of deny it will use ZREMRANGEBYSCORE and ZCARD commands in
+// one pipeline to remove old items in the list of hits.
+// In case of allow it will additionally use ZADD with a second
+// roundtrip.
```

Comment:
Lets keep previous comment, no need to draw reader attention and later tell them its deprecated :)